chessboard/script.js

```diff
@@ -0,0 +1,97 @@
+const board = document.querySelector(".board");
+const chess = document.querySelector(".chess");
+const bottom = document.createElement("div");
+const left = document.createElement("div");
+bottom.className = "bottom";
+left.className = "left";
+for (let i = 0; i < 8; i++) {
```

Comment:
While working with vanilla JS, it's always good to follow OOPS practices, utilize helper functions for modular code, and invoke these functions as needed